testsuite/pytests/test_compartmental_model.py

```diff
@@ -942,6 +942,16 @@ def test_add_assign(self):
         self.assertEqual(compartments[1]["comp_idx"], 1)
         self.assertEqual(compartments[1]["parent_idx"], 0)
 
+        cm2 = nest.Create('cm_default')
+        cm2.compartments = {"parent_idx": -1, "params": SP}
+        cm2.compartments += [{"parent_idx": 0, "params": DP[0]}, {"parent_idx": 0, "params": DP[0]}]
```

Comment:
Should the second `parent_idx` be 1 here? The test seems to pass regardless though.
```suggestion
        cm2.compartments += [{"parent_idx": 0, "params": DP[0]}, {"parent_idx": 1, "params": DP[0]}]
```

Reply:
It's just a different (valid) tree structure. You can put whichever one you prefer.